index.js

```diff
@@ -28,6 +29,11 @@ function ghGot(path, opts) {
 		opts.headers['content-length'] = 0;
 	}
 
+	if (isPlainObj(opts.body)) {
+		opts.headers['content-type'] = 'application/json; charset=utf-8';
```

Comment:
I don't charset is needed or even allowed here.

https://github.com/request/request/blob/290111d4e94ac9f6c8d3abecbc8920c96d1b32be/request.js#L1181
https://github.com/request/request/issues/383